lib/connection/connection_config.js

```diff
@@ -468,6 +469,15 @@ function ConnectionConfig(options, validateCredentials, qaMode, clientInfo)
     forceStageBindError = options.forceStageBindError;
   }
 
+  let disableQueryContextCache = false;
+  if (Util.exists(options.disableQueryContextCache)) {
+    
+    Errors.checkArgumentValid(Util.isBoolean(options.disableQueryContextCache),
+    ErrorCodes.ERR_CONN_CREATE_INVALID_DISABLED_QUERY_CONTEXT_CACHE);
```

Comment:
should this line be indented? it's part of the expression above

Reply:
Removed

Reply:
it does not seem to be fixed?

Reply:
I think it is a delay on Github. Now it is fixed.